cobigen/cobigen-core-parent/cobigen-core/src/main/java/com/devonfw/cobigen/impl/config/reader/TemplatesConfigurationReader.java

```diff
@@ -98,9 +98,24 @@
      */
     public TemplatesConfigurationReader(Path templatesRoot) throws InvalidConfigurationException {
```

Comment:
Are you sure, that you have to change anything here?
I would not think so as the context configuration reader will directly provide the root path, which already contains src/main/templates or not (if defined in the old style)

Reply:
I actually haven't thought of that, will test it